samples/show-extruded-features/src/main/java/com/esri/arcgismaps/sample/showextrudedfeatures/screens/ShowExtrudedFeaturesScreen.kt

```diff
@@ -0,0 +1,102 @@
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.SceneView
+import com.esri.arcgismaps.sample.showextrudedfeatures.components.ShowExtrudedFeaturesViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen containing a SceneView and simple UI to switch extrusion expressions.
+ */
+@Composable
+fun ShowExtrudedFeaturesScreen(sampleName: String) {
+    val viewModel: ShowExtrudedFeaturesViewModel = viewModel()
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = { paddingValues ->
+            Column(modifier = Modifier
+                .fillMaxSize()
+                .padding(paddingValues)) {
+
+                // SceneView composable from the toolkit renders the 3D scene
+                Box(modifier = Modifier
+                    .fillMaxSize()
+                    .weight(1f)) {
+                    SceneView(
+                        modifier = Modifier.fillMaxSize(),
+                        arcGISScene = viewModel.arcGISScene
+                    )
+                }
+
+                // Controls for switching the extrusion statistic
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(12.dp),
+                    horizontalArrangement = Arrangement.SpaceEvenly
+                ) {
+                    val current = viewModel.currentStatistic
+                    Button(
+                        onClick = { viewModel.updateExtrusionStatistic(ShowExtrudedFeaturesViewModel.Statistic.TotalPopulation) },
+                        enabled = current != ShowExtrudedFeaturesViewModel.Statistic.TotalPopulation
+                    ) {
+                        Text(text = ShowExtrudedFeaturesViewModel.Statistic.TotalPopulation.label)
+                    }
+
+                    Button(
+                        onClick = { viewModel.updateExtrusionStatistic(ShowExtrudedFeaturesViewModel.Statistic.PopulationDensity) },
+                        enabled = current != ShowExtrudedFeaturesViewModel.Statistic.PopulationDensity
+                    ) {
+                        Text(text = ShowExtrudedFeaturesViewModel.Statistic.PopulationDensity.label)
+                    }
+                }
+
+                // Show a message dialog if an error occurred in the ViewModel
+                viewModel.messageDialogVM.apply {
+                    if (dialogStatus) {
+                        MessageDialog(
+                            title = messageTitle,
+                            description = messageDescription,
+                            icon = Icons.Default.Info,
+                            onDismissRequest = ::dismissDialog
+                        )
+                    }
+                }
+            }
+        }
+    )
+}
```

Comment:
nitpick end of line at end of file.
```suggestion
}

```

Reply:
Added